teachopencadd/talktorials/019_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1250 @@
+{

```

Comment:
<p>This <code># NBVAL_CHECK_OUTPUT</code>  will almost certainly fail because the minimization is a stochastic process and we are drawing velocities from a distribution, so for sure we will <em>not</em> get the same exact numbers every time. </p><p> </p><p>Remove this comment and maybe, in a new cell, check if everything ran correctly by checking that the simulation objects has reached the desired number of steps.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
@Mika-Le

How about removing the #NBVAL_CHECK_OUTPUT from the simulation step cell and adding another cell afterwards with the following content:

`simulation_time = round(simulation.context.getState().getTime().value_in_unit(unit.femtoseconds))`
`if simulation_time == steps * 2:`
`print("Simulation finished successfully!")`
`else:`
`print("Simulation failed!")`
`# NBVAL_CHECK_OUTPUT`